src/buildHTML.js

```diff
@@ -581,166 +581,6 @@ groupTypes.genfrac = function(group, options) {
         options);
 };
 
-groupTypes.array = function(group, options) {
```

Comment:
Copied to `const htmlBuilder = function(group, options) { ...` in array.js.

Reply:
Thanks for leaving hints like this!